src/magic_enchantment.cpp

```diff
@@ -352,6 +352,11 @@ bool enchantment::is_monster_relevant() const
         }
     }
 
+    // check for hit you / me effects
+    if( hit_you_effect.size() > 0 || hit_me_effect.size() > 0 ) {
```

Comment:
Clang errors?

Reply:
Huh, I missed that for the other seemingly? unrelated errors.  Switched over to empty()